zhaquirks/tuya/ts0601_motion.py

```diff
@@ -398,6 +565,7 @@ class MmwRadarMotionGPP(CustomDevice):
             ("_TZE200_ar0slwnd", "TS0601"),
             ("_TZE200_sfiy5tfs", "TS0601"),
             ("_TZE200_mrf6vtua", "TS0601"),
+            ("_TZE204_sxm7l9xa", "TS0601"),
```

Comment:
("_TZE204_qasjif9e", "TS0601"), can be added here as well. 
I tested locally on my machine and the sensor  _TZE204_qasjif9e works perfectly with this quirk.

Reply:
Yeah.  @pgroene has been kind enough to provide [a separate PR](https://github.com/zigpy/zha-device-handlers/pull/2525) for that. 👍🏼